common/test/acceptance/pages/lms/discussion.py

```diff
@@ -380,14 +380,17 @@ def get_selected_sort_preference(self):
         """
         Return the text of option that is selected for sorting.
         """
-        options = self.q(css="body.discussion .forum-nav-sort-control option")
-        return options.filter(lambda el: el.is_selected())[0].get_attribute("value")
+        selected_value = self.browser.execute_script(
```

Comment:
@andy-armstrong this is to make it compatible with Chrome. Currently on chrome the previous code (and selenium code that I tried) returned List Index Out of Range, even with index [0] which on my understanding is because chrome browser and driver version is old (that we are using).